Description: newer avr-libc debricated some typedefs, this enables compatabillity
Author: Scott Howard <showard@debian.org>

Index: arduino/hardware/arduino/avr/platform.txt
===================================================================
--- arduino.orig/hardware/arduino/avr/platform.txt	2014-03-07 12:43:40.520144544 -0500
+++ arduino/hardware/arduino/avr/platform.txt	2014-03-07 12:44:39.108145937 -0500
@@ -14,12 +14,12 @@
 # Default "compiler.path" is correct, change only if you want to overidde the initial value
 #compiler.path={ide.path}/tools/avr/bin/..
 compiler.c.cmd=avr-gcc
-compiler.c.flags=-c -g -Os -w -ffunction-sections -fdata-sections -MMD
+compiler.c.flags=-c -g -Os -w -ffunction-sections -fdata-sections -MMD -D__PROG_TYPES_COMPAT__
 compiler.c.elf.flags=-Os -Wl,--gc-sections
 compiler.c.elf.cmd=avr-gcc
 compiler.S.flags=-c -g -x assembler-with-cpp
 compiler.cpp.cmd=avr-g++
-compiler.cpp.flags=-c -g -Os -w -fno-exceptions -ffunction-sections -fdata-sections -MMD
+compiler.cpp.flags=-c -g -Os -w -fno-exceptions -ffunction-sections -fdata-sections -MMD -D__PROG_TYPES_COMPAT__
 compiler.ar.cmd=avr-ar
 compiler.ar.flags=rcs
 compiler.objcopy.cmd=avr-objcopy
